src/browsers/index.ts

```diff
@@ -72,6 +74,40 @@ export class BrowserManager {
     return dataDirPath;
   };
 
+  public getVersionJSON = async () => {
+    if (!this.versionCache) {
+      const port = await getPort();
+      const config = this.config;
+      config.setPort(port);
```

Comment:
This will re-set the port for the docker container, which I don't think is intentional (plus there's a better way with routes to do this I reckon). Let's meetup and I can go over that with you